field.go

```diff
@@ -400,6 +400,15 @@ func Object(key string, val zapcore.ObjectMarshaler) Field {
 	return Field{Key: key, Type: zapcore.ObjectMarshalerType, Interface: val}
 }
 
+// Inline is similar to Object, but does not nest the object under a field
+// name, but adds the fields to the current namespace inline.
```

Comment:
This sentence reads weird with two "but"s. How about,

    // Inline constructs a Field that is similar to Object, but it
    // will add the elements of the provided ObjectMarshaler to the
    // current namespace.

Reply:
Didn't notice the two "but"'s, updated.